docs-chef-io/content/workstation/getting_started.md

```diff
@@ -14,290 +14,295 @@ aliases = ["/workstation_setup.html", "/chefdk_setup.html", "/workstation.html",
+  ```
 
-### Install a Code Editor
-
-A good visual code editor is not a requirement for working with Chef
-Infra, but a good code editor can save you time. A code editor should
-support the following: themes, plugins, snippets, syntax Ruby code
-coloring/highlighting, multiple cursors, a tree view of the entire
-folder/repository you are working with, and a Git integration.
-
-These are a few common editors:
+{{< /foundation_tabs_panel >}}
+{{< foundation_tabs_panel panel-id="ruby-env-windows-panel" >}}
+{{< warning >}}
 
-- [Visual Studio Code (free/open source)](http://code.visualstudio.com)
-- [GitHub Atom - (free/open source)](http://atom.io)
+  On Microsoft Windows, `C:/opscode/Chef Workstation/bin` must be before `C:/opscode/Chef Workstation/embedded/bin` in the `PATH`.
 
-Chef Infra support in editors:
+{{< /warning >}}
+{{< /foundation_tabs_panel >}}
 
-- [VSCode Chef Infra Extension](https://marketplace.visualstudio.com/items?itemName=chef-software.Chef)
-- [Chef on Atom](https://atom.io/packages/language-chef)
+{{< /foundation_tabs_panels >}}
 
-### Starter Kit
+## Setup Your Chef Repo
 
-If you have access to Chef Infra Server through Automate or Chef Manage,
-you can download the starter kit. The starter kit will create the
-necessary configuration files: the `.chef` directory, `config.rb`,
-`ORGANIZATION-validator.pem`, and `USER.pem`. Simply download the
-starter kit and then move it to the desired location on your Chef
-Workstation machine.
+{{% chef_repo_description %}}
 
-## Configure the Chef Repository
+If you're setting up Chef for the very first time **in your organization**, then you will need a Chef repository for saving your cookbooks and other work.
 
-### With WebUI
+Use the [chef generate repo]({{< relref "ctl_chef.md#chef-generate-repo" >}}) command to create your Chef repository. For example, to create a repository called
+`chef-repo`:
 
-Use the following steps to manually set up the chef-repo and to use the
-Chef management console to get the `.pem` and `config.rb` files.
+``` bash
+chef generate repo chef-repo
+```
 
-#### Get Config Files
+## Your Chef Workstation Installation
 
-For a Chef Workstation installation that will interact with the Chef
-Infra Server (including the hosted Chef Infra Server), log on and
-download the following files:
+Chef Workstation installs two different things onto your computer:
 
-- `config.rb`. This configuration file can be downloaded from the **Organizations** page.
-- `ORGANIZATION-validator.pem`. This private key can be downloaded from the **Organizations** page.
-- `USER.pem`. This private key can be downloaded from the **Change Password** section of the **Account Management** page.
+1. [Chef products and tools]({{< relref "install_workstation.md" >}})
+1. The `~/.chef` directory
 
-#### Move Config Files
+The `~/.chef` directory starts out with one file, which is the `config.rb` with example configuration settings.
```

Comment:
This isn't actually the file we create. This setup is referencing the old config.rb setup. Here.s the new files created by the Workstation app:

```
❰tsmith❙~/.chef❱✔≻ ls
credentials
❰tsmith❙~/.chef❱✔≻ cat credentials
# This is the Chef Infra credentials file used by the knife CLI and other tools
# This file supports defining multiple credentials profiles, to allow you to switch between users, orgs, and Chef Infra Servers.

# Example credential file configuration:
# [default]
# client_name = 'MY_USERNAME'
# client_key = '/Users/tsmith/.chef/MY_USERNAME.pem'
# chef_server_url = 'https://api.chef.io/organizations/MY_ORG'
```